Follow-up to PR#1926 for REST API Overview

adellape opened the pull request:
Follow-up to https://github.com/openshift/openshift-docs/pull/1926. Minor edits, mostly formatting.

@aweiteka @openshift/team-documentation PTAL? Pretty builds:

(OSE) http://file.rdu.redhat.com/~adellape/072916/openshift-enterprise/edit_api_1926/rest_api/

(AR) http://file.rdu.redhat.com/~adellape/072916/atomic-registry/edit_api_1926/rest_api/

@aweiteka I noticed this topic references `dev_guide/service_accounts`, which is not currently in AR's User Guide. Did you want it to be?

aweiteka commented:
> I noticed this topic references dev_guide/service_accounts, which is not currently in AR's User Guide. Did you want it to be?

Yes, it should be. Good catch. Do you want to add it in this PR or want me to do that?

aweiteka commented:
LGTM

adellape commented:
@aweiteka Doesn't look like the `dev_guide/service_accounts` topic did any problematic linking, so I've just added it in this PR. I've updated the preview links above. Here's the new topic:

http://file.rdu.redhat.com/~adellape/072916/atomic-registry/edit_api_1926/dev_guide/service_accounts.html

WDYT?

aweiteka commented:
> Doesn't look like the dev_guide/service_accounts topic did any problematic linking, so I've just added it in this PR.

+1. Looks good. Thank for adding it.

adellape commented:
@aweiteka On second thought, that `dev_guide/service_accounts` topic talks about pods, builds, and replication controllers in a few spots. Should those be ifdef'd out? How about the "Default Service Accounts and Roles" section in general?

aweiteka commented:
@adellape oh, right, I should have read this more carefully. These sections don't belong:

* "Managing allowed secrets"
* "Using a service account’s credentials inside a container"

The last sentence of the "Overview" is good. In "Managing service accounts" we should be using `oc create serviceaccount <sa_name>`. Want me to update?

adellape commented:
> Want me to update?

@aweiteka Yeah that'd be good. Thanks.
